scripts/ci

```diff
@@ -3,6 +3,13 @@ set -e
 
 cd $(dirname $0)
 
+if ./only-ui-bumps.sh; then
```

Comment:
We have caught quite a few issues while trying to release new RCs. Are we sure we want to do this?

Reply:
I don't think releasing RCs should be a medium to catch issues, currently we are wasting release captains time and it's causing RC's to be delayed while not even all issues that are found are even fixed. We can now move that to restarting the build that publishes the artifacts after the merge so the release process is quick and automated.

Reply:
Any thoughts on this @thedadams ? I can request more reviews if we want to get a broader opinion.

For me this is about making the release process quicker, the RCs are a snapshot of already validated code, there is no need to spend all this time and resources to re-validate what we already have. The issue with current CI is that its flakey, in my opinion not something that we should pin on the release process but we need to structurally improve and need to brainstorm about together. I'm actually in favor of dropping more time and resource consuming steps in the release process, mostly because they get restarted to get the release finished anyway.

Reply:
I agree with you about the need to improve. My point here was that there have been multiple times in release processes recently when a bug was discovered when tagging a new RC or bumping UI tags. I agree that this is not desirable, but, until the things you have pointed our are improved, it is valuable. In my opinion, valuable enough to keep running tests on UI bumps and tags until things can be improved.

This is only my opinion. I am happy to give way to other opinions if I am in the minority.

Reply:
Would you accept this change if we would run tests on a regular schedule to find issues to unlink it from the (pre) release process? Also, like I said before, the value of this is minimal as there is only value if the person creating the tag/PR goes investigating, not if that person just restarts til it works.